specs/engage-digital_openapi3.yaml

```diff
@@ -5146,6 +5146,184 @@ paths:
       summary: Getting all timezones
       tags:
         - Timezones
+  /user_capacities:
+    get:
+      description: >-
+        This method renders all user capacities ordered by creation date (ascending).
+
+        Authorization​: only users who can manage topology.
+      operationId: getAllUserCapacities
+      parameters:
+        - description: The record index to start. Default value is 0.
+          in: query
+          name: offset
+          required: false
+          schema:
+            format: int32
+            type: integer
+        - description: >-
+            The max number of records to return. Default value is 30, max value
+            is 150.
+          in: query
+          name: limit
+          required: false
+          schema:
+            format: int32
+            type: integer
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+               $ref: '#/components/schemas/GetAllUserCapacities'
+          description: Success
+      summary: Getting all User Capacities
+      tags:
+        - UserCapacity
+    post:
+        description: >-
+          This method creates a new user capacity. In case of success it renders the
+          updated record, otherwise, it renders an error (422 HTTP code).
+          Please note that the order of the parameters is important, as the default and
+          max capacities will apply to the immediately above channel_id.
+
+          Authorization​: only users who can manage topology.
+        operationId: createUserCapacity
+        parameters:
+          - in: path
+            name: userCapacityId
+            required: true
+            schema:
+              type: string
+          - description: User Capacity name.
+            in: query
+            name: name
+            required: false
+            schema:
+              type: string
+          - description: The channel id on which apply the capacity.
+            in: query
+            name: channels[][channel_id]
+            required: false
+            schema:
+              type: string
+          - description: the default capacity to apply on the previous channel id.
+            in: query
+            name: channels[][default_capacity]
+            required: false
+            schema:
+              type: integer
+          - description: the maximum capacity to apply on the previous channel id.
+            in: query
+            name: channels[][max_capacity]
+            required: false
+            schema:
+              type: integer
+        responses:
+          '200':
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/UserCapacity'
+            description: Success
+        summary: Creating a user capacity
+        tags:
+          - UserCapacity
+  '/user_capacities/{userCapacityId}':

```

Comment:
IMO we should display the routes in this order:

- GET /user_capacities
- GET /user_capacities/:id
- POST /user_capacities
- PUT /user_capacities/:id
- DELETE /user_capacities/:id

Reply:
Actually I don't think it is possible since they need to be grouped by url.

Reply:
oh okay then :+1:

Reply:
And actually on other endpoints the order seems to be on alphabetical order : 

```
- GET      /user_capacities
- POST     /user_capacities
- DELETE   /user_capacities/:id
- GET      /user_capacities/:id
- PUT      /user_capacities/:id
```

I switched it to match this.